src/Altinn.App.Core/Internal/Expressions/ExpressionEvaluator.cs

```diff
@@ -125,7 +125,7 @@ public static bool EvaluateBooleanExpression(LayoutEvaluatorState state, Compone
             throw new NotImplementedException("Component lookup for components in groups not implemented");
         }
 
-        if (!targetContext.Component.DataModelBindings.TryGetValue("simpleBinding", out var binding))
+        if (targetContext.Component is not BaseComponent || !targetContext.Component.DataModelBindings.TryGetValue("simpleBinding", out var binding))
```

Comment:
Jeg ville gått for `is not null`, siden jeg leser dette som et hint om at den kan være noe annet en BaseComponent.

Ellers er det vel mulig å skrive
```suggestion
        if (targetContext.Component?.DataModelBindings.TryGetValue("simpleBinding", out var binding) != true)
```
siden `..?.DataModelBindings.TryGetValue` kan være `null`, `true` eller `false` og men alt annet enn true skal trigge feilmelding.

Reply:
Jepp det var faktisk det jeg gjorde først, kan ta det tilbake. Funker like bra det